packages/docz-theme-default/src/components/ui/Render/index.tsx

```diff
@@ -388,13 +388,13 @@ export class Render extends Component<RenderComponentProps, RenderState> {
 
   private transformCode(code: string): string {
     return `
-      const App = ({ children }) => (
+      const _App = ({ children }) => (
         <React.Fragment>
           {children && typeof children === 'function' ? children() : children}
         </React.Fragment>
       )
 
-      render(<App>${code}</App>)
+      render(<_App>${code}</_App>)
```

Comment:
```suggestion
      render(<_Docz_App>${code}</_Docz_App>)
```

Reply:
I think that use underscore is something weird, I prefer `DoczApp` like normal!

Reply:
@pedronauck I renamed it do `DoczApp`